### PR TITLE
Display Commit Id on Frontend

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**
fixes https://github.com/lfortran/lcompilers_frontend/issues/2.

This uses the approach shared in https://github.com/lfortran/lcompilers_frontend/issues/2#issuecomment-1325723254.

### CODE REVIEW

**Comment by Shaikh-Ubaid** [comment]
>Your site is deployed at https://lfortran.github.io/pull_request_preview

The preview seems to work.

**Comment by Shaikh-Ubaid** [comment]
This is ready. Please possibly review and please share feedback.

File: components/MyFooter.js [hunk 1]
```diff
@@ -11,7 +12,7 @@ function MyFooter() {
                 width: "100%",
             }}
         >
-            <a href="https://lfortran.org/">LFortran</a> by <a href="https://lcompilers.org/">LCompilers</a>
+            <a href="https://lfortran.org/">LFortran</a> ({commit.id}) by <a href="https://lcompilers.org/">LCompilers</a>
```
Comment (certik) [inline components/MyFooter.js:15]: How about this:
```suggestion
            <a href="https://lfortran.org/">LFortran</a> (<a href="https://github.com/lfortran/lfortran/commit/{commit.id}">{commit.id}</a>) by <a href="https://lcompilers.org/">LCompilers</a>
```
I tested that the shorter hash still works in the url. I didn't test the PR with this change.

**Review by certik — APPROVED** [review]
Looks great, thanks for fixing it!

I provided one suggestion above.

**Comment by certik** [comment]
Ha, the link doesn't work, the id didn't get substituted. :)

**Comment by Shaikh-Ubaid** [comment]
Thank you for the suggestion and guidance, Dr. Ondrej. The commit-id link in the site preview seems to work.

**Comment by Shaikh-Ubaid** [comment]
I am merging this as it seems approved. After merging this. I will closely follow if the CI succeeds in the main branch and if the site at `dev.lfortran.org` works.

**Comment by Shaikh-Ubaid** [comment]
The CI succeeds and the site at https://dev.lfortran.org/ seems to work fine.

**Comment by certik** [comment]
Awesome, thank you!

**Pull request merged**